examples/basic/areas/block.py

```diff
@@ -1,3 +1,13 @@
+"""
+Creates a block plot using Bokeh and displays it using the show() function.
+
+.. bokeh-example-metadata::
+    :sampledata: None
+    :apis: bokeh.plotting.figure, bokeh.plotting.show, bokeh.models.Block
+    :refs: :ref:`uuserguide_plotting_visualization_glyphs`
```

Comment:
The references are a little obscure, they need to match the ones that exist in the docs source files, in this case:
```suggestion
    :refs: :ref:`ug_basic_areas_rects`
```